Example/Example-iOS/AnalyticsImplementationDemo.swift

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2020-Present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import Foundation
+import OktaLoggerAnalytics
+import AppCenterAnalytics
+
+class AppCenterAnalyticsManager: AnalyticsProviderProtocol {
```

Comment:
why is this class not implementing AnalyticsProvider protocol with track event call, can we reduce number of protocols that a provider has to implement ?

Reply:
provider has to implement only `AnalyticsProviderProtocol ` protocol @umangshah-okta